ui/tests/acceptance/secrets/backend/kv/kv-v2-workflow-create-test.js

```diff
@@ -29,7 +29,9 @@ module('Acceptance | kv-v2 workflow | secret and version create', function (hook
     this.store = this.owner.lookup('service:store');
     await login();
     await runCmd(mountEngineCmd('kv-v2', this.backend), false);
-    await writeVersionedSecret(this.backend, 'app/first', 'foo', 'bar', 2);
+    const secretPath = 'app/first_secret';
```

Comment:
added test coverage before fixing the api service to ensure the change did resolve the issue in the app